dotnet/src/webdriver/WebDriver.cs

```diff
@@ -679,7 +679,9 @@ protected void StartSession(ICapabilities capabilities)
 
             ReturnedCapabilities returnedCapabilities = new ReturnedCapabilities(rawCapabilities);
             this.capabilities = returnedCapabilities;
-            this.sessionId = new SessionId(response.SessionId);
+
+            string sessionId = response.SessionId ?? throw new WebDriverException("StartSession did not return a sessionId");
```

Comment:
According to spec, StartSession always contains a `sessionId`

![image](https://github.com/user-attachments/assets/18a66ba4-176b-43f0-b011-a0182232ba4f)

Reply:
It is reasonable.

Reply:
Can we improve "StartSession did not return a sessionId"? who is StartSession, who is "sessionId"?

Reply:
More human readable: "The ID of session not found when started new session.". Or "Remote end didn't respond with ID of session when required.". Please be more closer to users.

Reply:
I adjusted.